.cirrus.yml

```diff
@@ -4,55 +4,184 @@ env:
 BUILD_TEST_TASK_TEMPLATE: &BUILD_TEST_TASK_TEMPLATE
   arch_check_script:
     - uname -am
-  test_script:
+  prebuild_script:
+    - node --version
```

Comment:
Pulling these out into a script is a nice way to make it easy to migrate between different CI systems (which we've had to do a few times with Pact)

Reply:
agreed! this was my first pass so will script it up now i know it all works :)